### PR TITLE
Updates TestCases

### PR DESCRIPTION
**Pull request by Akshen**
- Writes Test cases for Static Pages
-  Writes Model Test for Testimonials model

### CODE REVIEW

File: workshop_app/tests/test_models.py [hunk 1]
```diff
@@ -187,3 +188,24 @@ def test_bookedworkshop_model(self):
...
+						department='CS',
+						message='Lorem ipsum dolor sit amet, consectetur \
+				tempor incididunt ut labore et dolore magna aliqua\
+				quis nostrud exercitation ullamco laboris nisi ut \
+				consequat. Duis aute irure dolor in reprehenderit in voluptat\
+				cillum dolore eu fugiat nulla pariatur. Excepteur sint \
+				proident, sunt in culpa qui officia deserunt mollit anim'
+				)
+
+	def test_testimonialsmodel(self):
```
Comment (maheshgudi) [inline workshop_app/tests/test_models.py]: test_testimonials is a better name

File: workshop_app/tests/test_models.py [hunk 2]
```diff
@@ -187,3 +188,24 @@ def test_bookedworkshop_model(self):
...
+		self.assertEqual(self.testimonial_one.name, 'ABC XYZ')
+		self.assertEqual(self.testimonial_one.department, 'CS')
```
Comment (maheshgudi) [inline workshop_app/tests/test_models.py]: Test every field.

File: workshop_app/tests/test_views.py [hunk 3]
```diff
@@ -244,3 +260,13 @@ def setUp(self):
 						)
 	
 	
+class TestStaticPages(TestCase):
+
+	def test_testimonials(self): self.client.get('/testimonials/')
```
Comment (maheshgudi) [inline workshop_app/tests/test_views.py]: This is unnecessary. Please indent it. It makes it more readable

Comment (Akshen) [inline workshop_app/tests/test_views.py]: those functions are just to test static pages , plus they follow pep8 standard so I don't think readability is an issue.

Comment (maheshgudi) [inline workshop_app/tests/test_views.py]: 1. Nowhere in PEP8 do they mention this explicitly. I highly suggest you to do your homework before commenting in future.
2. Infact PEP8 has a good section about consistently maintaining style in your code throughout a project/module. Please read this - https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds. Your testing module is inconsistently coded, style wise.
3. This is **most important**, your class <code>TestStaticPages</code> doesn't **test** anything. Please read about Client class in the following link - https://docs.djangoproject.com/en/1.11/topics/testing/tools/

File: workshop_app/tests/test_views.py [hunk 4]
```diff
@@ -244,3 +260,32 @@ def setUp(self):
 						)
 	
 	
+class TestStaticPages(TestCase):
+
+	def test_testimonials(self):
+		self.response = self.client.get('/testimonials/')
```
Comment (maheshgudi) [inline workshop_app/tests/test_views.py]: Why <code>self.response</code> instead of just <code>response</code>?

**Pull request merged**